src/main/java/com/ays/backend/user/exception/RoleNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.ays.backend.user.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
```

Comment:
please remove unused import.

Reply:
I just removed unnecessary import from the class.